app/src/main/cpp/switch/kernel/service.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include "../nce.h"
+
+namespace lightSwitch::kernel::service {
```

Comment:
i've said this a few times now so i wont repeat after this, change namespace here and in the rest of the files